### PR TITLE
adding windows build action

### PR DESCRIPTION
**Pull request by brettmc**
* build for windows
* add a short doc point linking to Windows build instructions
* rename linux build

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,6 +36,10 @@ Via pecl/pickle:
 install-php-extensions opentelemetry[-beta|-stable|-latest]
 ```
 
+### Windows
+
+See https://wiki.php.net/internals/windows/stepbystepbuild_sdk_2#building_pecl_extensions_with_phpize
```
Comment (pdelewski) [inline README.md:41]: Have not tried that for some time, however I'm not sure if this instruction is valid

**Review by pdelewski — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (brettmc) [reply README.md:41]: That's what the github action does, and it seems to work (compiles, and the tests all run and pass). I copied most of the steps from phpredis extension's windows build.

**Comment by wantedxnn** [comment]
@brettmc I think you chosen wrong settings ...
PHP8 still is builded with vs16 (VS 2019) so in matrix setup need to use 'windows-2019' because extension will can not be loaded by php.net builds ....

**Comment by brettmc** [comment]
> PHP8 still is builded with vs16 (VS 2019) so in matrix setup need to use 'windows-2019' because extension will can not be loaded by php.net builds

ok, changed to 2019.

> Also i suppose want to upload windows build artifact to pecl.php.net ..... if not then it's not a problem

I didn't know you could even do that. It looks like windows.php.net provides builds of many extensions, is that how it's usually done? I couldn't see anything relevant in the package upload section of pecl, which is the only bit I'm familiar with.

**Comment by wantedxnn** [comment]
@brettmc If remember you can also upload artifact build using 'actions/upload-artifact' from 'https://github.com/cmb69' .... but im not 💯 sure ....

**Comment by brettmc** [comment]
> you can also upload artifact build using 'actions/upload-artifact'

I'll create an issue to look at this. Upload artifact action will store it in the github run, but that's something.

**Pull request merged**